src/react-components/room/useMicrophone.js

```diff
@@ -23,29 +31,98 @@ export function useMicrophone(scene, updateRate = 50) {
         movingAvgRef.current.push(Date.now(), analyser.volume);
         const average = movingAvgRef.current.movingAverage();
         const nextVolume = max === 0 ? 0 : average / max;
-        setVolume(prevVolume => (Math.abs(prevVolume - nextVolume) > 0.05 ? nextVolume : prevVolume));
+        setMicVolume(prevVolume => Math.max(0.15, Math.abs(prevVolume - nextVolume) > 0.05 ? nextVolume : prevVolume));
         timeout = setTimeout(updateMicVolume, updateRate);
       };
 
       updateMicVolume();
```

Comment:
Was expecting to find useVolumeMeter here. Probably not really a big deal but I think its the same code (and if it isn't it might be worth consolidating)

Reply:
Nice, that was the idea behind the `useVolumeMeter` hook but I missed that for some reason, thanks.